lib/Charts/LineChart.js

```diff
@@ -287,16 +299,18 @@ function render(element, state) {
     if (state.mini) {
         yAxis.tickSize(0, 0);
     }
-    yAxisElements
-        .transition('1st').duration(transitionDuration)
+    const yAxisElementsElements = g.select('.y.axes').selectAll('.y.axis');
+
+    yAxisElementsElements
+        .transition(t)
         .attr('transform', (d, i) => ('translate(' + (- i * Size.yAxisWidth) + ', 0)'));
 
-    yAxisElements[0].forEach(yAxisElement => {
-        const yAxisD3 = d3.select(yAxisElement);
+    yAxisElementsElements.nodes().forEach(yAxisElement => {
```

Comment:
Thanks for cleaning up all my `[0]` stuff with `.nodes()` - that looks much more professional!